### PR TITLE
Stinking Cloud

### PR DESCRIPTION
**Pull request by drkekyll**
The vanilla spell also blocked actions instead of expending an action point preventing the use of Action Surge, so that's also fixed.

Also, the vanilla spell blocked "EyeStalkActionPoint" so I had the status expend one of those, but I believe whatever uses those has 2 of them, so I'm not sure if you want to have it spend both. I haven't encountered whatever it is in gameplay, so I have nothing to inform my opinion.

### CODE REVIEW

**Review by FelipeRenault — COMMENTED** [review]
I won't be able to test or modify it right now, but it is needed to include the root template on the list of files to unload if the option is disabled.
If you want to see attempt to do it, feel free to include a commit changing the lua files. Otherwise, I'll make the necessary change when I'm back home

**Comment by drkekyll** [comment]
I was still trying to parse how you implemented the modularity, but if I'm understanding correctly, the piece I was missing happens in BootstrapModule.lua. So, if I actually understand this properly, I only need to make a change to the default .json file?

**Comment by FelipeRenault** [comment]
Yes, that will be the only step required! I'm sorry I couldn't give you the file names and everything right away, since I'm travelling right now 😅

**Comment by drkekyll** [comment]
All good! It was a learning experience!

File: RAW/Public/RAW/RootTemplates/Source/Stinking_Cloud.lsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+					<attribute id="Description" type="TranslatedString" handle="ha65bdbbdge8f4g407fg86e1g78094f48a5b5" version="4" />
+					<attribute id="Icon" type="FixedString" value="Surface_Stinking" />
+					<attribute id="DecalMaterial" type="FixedString" value="99bbc5f7-647e-2c42-c057-b5ffd201bd58" />
+					<attribute id="MaterialType" type="uint8" value="21" />
+					<attribute id="SurfaceCategory" type="uint8" value="12" />
+					<attribute id="AiPathColor" type="fvec3" value="1 1 0" />
+					<attribute id="Summon" type="FixedString" value="bd8d5043-5cf7-427c-8dfc-829b7c6065d2" />
+					<attribute id="RollConditions" type="LSString" value="" />
+					<attribute id="ObscuredStateOverride" type="uint8" value="3" />
+					<attribute id="OnEnterDistanceOverride" type="float" value="" />
```
Comment (FelipeRenault) [inline RAW/Public/RAW/RootTemplates/Source/Stinking_Cloud.lsx:30]: Vanilla has this value set to 2. Do you happen to know what was the reason for it?

**Pull request merged**